recipe/bld.bat

```diff
@@ -1,7 +1,7 @@
 mkdir "%SRC_DIR%"\cpp\build
 pushd "%SRC_DIR%"\cpp\build
 
-cmake -G "%CMAKE_GENERATOR%" ^
+cmake -G "NMake Makefiles" ^
```

Comment:
I suggest using `jom` or `ninja` to make the build parallel.

Reply:
Is `ninja` working nowadays? We had also issues with it in Azure before...

Reply:
Yes. When did you have issues? I fixed some issues with Ninja several months ago.

Reply:
No longer any issues, thanks!